### ISSUE TITLE
Harvested items are refreshed after conversion

### ISSUE THREAD

**Issue by Acru** [FINDarkside/TLD-Save-Upgrader#1]
I realize I'm a bit late to start reporting bugs for this old niche project, but I was wanting to get back into TLD where my sandbox saves are from version 0.365, and I want to try to continue where I left off rather than starting over, for now.

This converter updates my old saves except every animal ever killed and harvested, and every resource harvested is refreshed, so there are freshly dead corpses all over the place.

Are you willing to give any assistance in fixing this, or am I on my own?

**Comment by FINDarkside** [comment]
Yeah I remember someone mentioning this years ago. Unfortunately you're on your own, but thanks for creating the issue anyway 😄

**Comment by Acru** [comment]
Alright, thanks for the quick reply at least, heh~

**Comment by Acru** [comment]
I pushed a fix, but thought I would also mention here that other harvestables in Mystery Lake region seem to be refreshing due to changes in UUIDs related to the update to the region, so probably not fixable~
